src/commands/uplink/NewEvent.ts

```diff
@@ -0,0 +1,158 @@
+import Command from '../../Command.js';
+import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
+import * as Events from '../../constants/events.js';
+
+
+const COMMAND_ID = 0x15;
+const COMMAND_TITLE = 'NEW_EVENT';
+// connect/disconnect events are biggest
+const COMMAND_BODY_MAX_SIZE = 6;
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface EventBase {}
+
+type EventTime = EventBase & {
+    time: number
+};
+
+type EventBatteryAlarm = EventBase & {
+    voltage: number
+};
+
+type EventActivateMtx = EventTime & {
+    mtxAddr: number
+};
+
+type EventConnection = EventBase & {
+    channel: number,
+    value: number
+};
+
+type EventMtx = EventBase & {
+    status1: number,
+    status2: number
+};
+
+interface INewEventParameters {
+    id: number,
```

Comment:
```suggestion
    eventId: number,
```
?

Reply:
replaced "eventData" with "data" property, in that case we must use for eventSequenceNumber instead of sequenceNumber